codegen/module_system.go

```diff
@@ -639,7 +648,10 @@ func (g *EndpointGenerator) Generate(
 			)
 		}
 	}
-	return ret, nil
+	return &BuildResult{
+		Files: returnFiles,
+		Spec:  nil,
```

Comment:
Would it be a good idea to return spec (if none-nil) for every instance? This way the clients/endpoint initializer module that depends on all clients/endpoints would have the specs right off the shelve.